services/scabbard/cli/src/main.rs

```diff
@@ -86,25 +86,24 @@ fn run() -> Result<(), CliError> {
         app = app.subcommand(
             SubCommand::with_name("contract")
                 .about("List, show, or upload a Sabre smart contract")
+                .setting(AppSettings::SubcommandRequiredElseHelp)
                 .subcommand(
                     SubCommand::with_name("upload")
                         .about("Upload a Sabre contract")
                         .args(&[
                             Arg::with_name("scar")
                                 .long_help(
-                                    "The name and version requirement of the .scar to upload, in \
-                                     the format 'name:version_req'. The contract name must not \
-                                     include underscores, '_'. The version requirement can \
-                                     be any valid semver requirement string.",
+                                    "Name and version requirement of the smart contract archive \
+                                     (scar) file to upload",
                                 )
                                 .required(true),
-                            Arg::with_name("path")
+                            Arg::with_name("paths")
```

Comment:
"path" singular is more common for a list of directories to search (for example, $PATH, $LD_LIBRARY_PATH, etc.)

Reply:
Dropped the commit